helix-term/src/commands.rs

```diff
@@ -3587,7 +3587,7 @@ pub fn completion(cx: &mut Context) {
     use helix_core::chars;
     let mut iter = text.chars_at(cursor);
     iter.reverse();
-    let offset = iter.take_while(|ch| chars::char_is_word(*ch)).count();
+    let offset = iter.take_while(|ch| chars::char_is_whitespace(*ch)).count();
```

Comment:
This seems to be the reverse of what you want: you want to scan backwards while char is **not** whitespace

Reply:
@archseer doing `!chars::char_is_whitespace` seems to break `.` and `::` apparently. 
Another solution proposed @ChrHorn  `chars::char_is_word(*ch) || *ch == '\\'` which solves this particular problem.
I guess we can go along with that for now?
It would be nice if we could define this logic for the Julia LSP specifically so as to not break other langauges